gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2-entrypoints/step-2-entrypoints-0-architecture.component.ts

```diff
@@ -60,7 +60,7 @@ export class Step2Entrypoints0ArchitectureComponent implements OnInit, OnDestroy
     const currentStepPayload = this.stepService.payload;
 
     this.entrypointService
-      .v4ListSyncEntrypointPlugins()
+      .v4ListSyncEntrypointPlugins(['icon'])
```

Comment:
suggestion: set the `expand` query param directly in the service, to avoid to repeat it in the components.